sdk/formrecognizer/Azure.AI.FormRecognizer/src/Models_internal.cs

```diff
@@ -3,7 +3,7 @@
 
 using Azure.Core;
 
-namespace Azure.AI.FormRecognizer.Custom
+namespace Azure.AI.FormRecognizer.Training
```

Comment:
Would it make more sense if these `something_internal` types were in the `Models` namespace? @annelo-msft ?

Reply:
Thanks Caio!
I started a conversation on our Teams channel about it.